test/integrations/destinations/ga4/dataDelivery/business.ts

```diff
@@ -0,0 +1,351 @@
+    },
+  },
+  {
+    id: 'ga4_v1_scenario_2',
+    name: 'ga4',
+    description:
+      '[Proxy v1 API] :: Test for a invalid event name - where the destination responds with 200 with error for invalid event name',
+    successCriteria: 'Should return 200 with error with destination response',
+    scenario: 'Business',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            headers,
+            params,
+            JSON: invalidEventNameRequest,
+            endpoint: 'https://www.google-analytics.com/debug/mp/collect',
+          },
+          metadataArray,
+        ),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            message:
+              'Validation Server Response Handler:: Validation Error for ga4 of field path :events | NAME_INVALID-Event at index: [0] has invalid name [campaign@details]. Only alphanumeric characters and underscores are allowed.',
+            response: [
+              {
+                error:
+                  'Validation Server Response Handler:: Validation Error for ga4 of field path :events | NAME_INVALID-Event at index: [0] has invalid name [campaign@details]. Only alphanumeric characters and underscores are allowed.',
+                metadata: generateMetadata(1),
+                statusCode: 400,
+              },
+            ],
+            statTags: expectedStatTags,
+            status: 400,
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'ga4_v1_scenario_3',
+    name: 'ga4',
+    description:
+      '[Proxy v1 API] :: Test for a invalid parameter value - where the destination responds with 200 with error for invalid parameter value',
+    successCriteria: 'Should return 200 with error with destination response',
+    scenario: 'Business',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            headers,
+            params,
+            JSON: invalidParameterValueRequest,
+            endpoint: 'https://www.google-analytics.com/debug/mp/collect',
+          },
+          metadataArray,
+        ),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            message:
+              'Validation Server Response Handler:: Validation Error for ga4 of field path :undefined | INTERNAL_ERROR-Validation of item.price should prevent conversion from unsupported value [string_value: "$19"]',
+            response: [
+              {
+                error:
+                  'Validation Server Response Handler:: Validation Error for ga4 of field path :undefined | INTERNAL_ERROR-Validation of item.price should prevent conversion from unsupported value [string_value: "$19"]',
```

Comment:
repeated

Reply:
no, it's not repeated. error type is different. one is for invalid event name and other is for invalid parameter

Reply:
![image](https://github.com/rudderlabs/rudder-transformer/assets/33080863/b724563c-3bb6-4920-bb42-ab7431b254d5)

Reply:
is it possible keep them as variables

Reply:
addressed